packages/app-accounts/src/Editor.tsx

```diff
@@ -15,6 +15,8 @@ import accountObservable from '@polkadot/ui-keyring/observable/accounts';
 import withObservableBase from '@polkadot/ui-react-rx/with/observableBase';
 
 import AddressSummary from '@polkadot/ui-app/AddressSummary';
+import UploadButton from './UploadButton';
```

Comment:
Blank line before. Remove blank line before preceding import (which is in the same library block)